tests/format/markdown/code/typescript-trailing-comma.md

```diff
@@ -0,0 +1,3 @@
+```typescript
```

Comment:
Better put two  blocks in one file.

Reply:
Maybe "TypeScript" lang should also tested?

Reply:
No, we don't format `TypeScript`:

**Prettier 3.0.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEADdAVAngBzgZTACcBLHGAHSgEYACegalrvtoG4r1UrP0Zc4AZ2JlKNVkxb0OULiAA0ICORLRByUAEMiRCAHcACtoTqUmgDZ7NWdYoBGRTWADWcGPk0BbOABkSUOGQAMwtBOHtHFzd8HCd-AHNkGCIAV3CQMM8SJNT0wQTzOABFFIh4YND0gCtBAA98AuLS8qQQ8zDFAEdmuANdHFMQTUEAWgC4ABNJhRBkzRJzBIBhCE9PTWQh83MZ-Kh4woBBGGSSOxT4AzgiPwCK9vSACxhPcwB1R5J4QViwAhMviQAG5fLCbMCCWwgIFpACSUCmsEIpHIhwR+H4hXuHRAOF0YTejhwmzxQmuQMCin8YSIMD6mni62x6ViRBpm3WRGcE30UBmeP8MDeJAmMEeyAAHAAGRREODdEhy+mMjatSqKGCaOzC0XipAAJkUKTCGC1pjaOLgnjskymEx8mn2KQZcAAYhAiOsTglNpoLhAQABfQNAA)
<!-- prettier-ignore -->
```sh
--parser markdown
```

**Input:**
<!-- prettier-ignore -->
````markdown
```TypeScript
1   + 1    ;
```

```typescript
1   + 1    ;
```
````

**Output:**
<!-- prettier-ignore -->
````markdown
```TypeScript
1   + 1    ;
```

```typescript
1 + 1;
```

````